backend/backend/settings/base.py

```diff
@@ -396,10 +396,6 @@ def get_required_setting(
             "PASSWORD": f"{DB_PASSWORD}",
             "PORT": f"{DB_PORT}",
             "ATOMIC_REQUESTS": ATOMIC_REQUESTS,
-            "OPTIONS": {
-                "options": f"-c search_path={DB_SCHEMA}",
-                "application_name": os.environ.get("APPLICATION_NAME", ""),
```

Comment:
@muhammad-ali-e why was this removed? @Deepak-Kesavan had recently added this to know which service made the request

Reply:
@chandrasekharan-zipstack We are not using the options in pgbouncer initial conention.
@Deepak-Kesavan Is it needed?.